documentation/inventory_file_v2.md

```diff
@@ -178,751 +178,6 @@ targets:
-
-```yaml
-plugins:
-  pkcs7:
-    private-key: ~/bolt_private_key.pem
-```
-
--   `keysize`: They size of the key to generate with `bolt secret createkeys`. The default is `2048`
--   `private-key`: The path to the private key file. The default is `keys/private_key.pkcs7.pem`
--   `public-key`: The path to the public key file. The default is `keys/public_key.pkcs7.pem`
-
-**Vault plugin**
-
-This plugin allows configuration values to be set by accessing secrets from a key/value engine on a Vault server. It supports several fields:
-
--   `_plugin`: The value of `_plugin` must be `vault`
--   `server_url`: The URL of the Vault server (optional, defaults to `ENV['VAULT_ADDR']`)
--   `auth`: The method for authorizing with the Vault server and any necessary parameters (optional, defaults to `ENV['VAULT_TOKEN']`)
--   `path`: The path to the secrets engine (required)
--   `field`: The specific secret being used (optional, defaults to a Ruby hash of all secrets at `path`)
--   `version`: The version of the key/value engine (optional, defaults to 1)
--   `cacert`: The path to the CA certificate (required when using `https`, defaults to `ENV['VAULT_CACERT']`)
-
-**Authentication methods**
-
-Vault requires a token to assign an identity and a set of policies to a user before accessing secrets. The Vault plugin offers two authentication methods:
-
--   **Token-based**. Authenticate using a token. This method requires the following fields:
-    -   `method`: The value of `method` must be `token`
-    -   `token`: The token to authenticate with
--   **Username- and password-based**. Request a token by logging into the Vault server with a username and password. This method requires the following fields:
-    -   `method`: The value of `method` must be `userpass`
-    -   `user`: The username
-    -   `pass`: The password
-
-You can add any Vault plugin field to the inventory configuration. For example, this is how you would access the `private-key` secret on a Vault key-value engine (version 2) mounted at `secrets/bolt`:
-
-```yaml
-version: 2
-targets:
-  - ...
-config:
-  ssh:
-    user: root
-    private-key:
-      key-data:
-        _plugin: vault
-        server_url: http://127.0.0.1:8200
-        auth:
-          method: userpass
-          user: bolt
-          pass: bolt
-        path: secrets/bolt
-        field: private-key
-        version: 2
-```
-
-You can also set configuration values in the config file (typically `bolt.yaml`) under the `plugins` field. If a field is set in both the inventory file and the config file, Bolt uses the value set in the inventory file. The available fields for the config file:
-
--   `server_url`
--   `cacert`
--   `auth`
-
-```yaml
-plugins:
-  vault:
-    server_url: https://127.0.0.1:8200
-    cacert: /path/to/cert
-    auth:
-      method: token
-      token: <XXXX-XXXX>
-```
-
-## Inventory config
```

Comment:
The rest of this is duped from inventory v1 not the plugin docs right?

Reply:
https://puppet.com/docs/bolt/latest/inventory_file.html#inventory-config Yes - should I still have a section in this doc that links to that? That seems useful - I'll update